lib/management/push.js

```diff
@@ -277,7 +277,7 @@ function addOverlayMessageListener(callback) {
 
 function startPushConnectionHealthCheckServer(cb) {
     const server = http.createServer((req, res) => {
-        if (req.url !== '/_/healthcheck') {
+        if (req.url !== '/healthcheck') {
```

Comment:
not sure this is related.
I don't know what this "push connection" is, but it seems to be a dedicated server : so it is not needed for our purpose I think, and changing it here would break the other end of the push channel?

Reply:
it seems to be a server that has listent to a websocket and pushes some metrics somewhere. it has the healthcheck route and I assumed that route is checked. if it is not, we could leave it here untouched.

Reply:
i am not sure, but seeing the same value is not proof enough : since this `/_/healthcheck` is used everywhere...
hopefully someone knowledgeable on the subject will be able to confirm either way....

Reply:
right, wait for confirmation from someone else then regarding the outcome I will push a new commit.
I'll try to limit the number of commits as it is very hard to make them land and have a green build

Reply:
After looking around, this piece of code is not called by cloudserver. it is a dedicated server that can be run manually (manually => yarn run start_metrics_server) may be it is used by other components or simply unused at all.

Reply:
I think I should update the route to makes sure it matches the same route name as expected by zenko in general

Reply:
i confirmed with Rached that this is not used in Artesca (and should generally be removed, it was related to Orbit and the "cloud management" feature or Zenko 1)

Reply:
I am not too sure, this is run in artesca, or at least when I run a local cluster I have a pod running, I will try on a fresh artesca, but I think it runs. we could update it so it uses the `/live` route even though it will be removed for now it is here and should follow the update ?